src/promptflow/tests/test_configs/flows/basic_with_builtin_llm_node/storage_record.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
how is the reord file corresponding to a flow/node run?

Reply:
offline syncd